pkg/services/ngalert/state/manager.go

```diff
@@ -182,6 +198,9 @@ func (st *Manager) setNextState(ctx context.Context, alertRule *ngModels.AlertRu
 	case eval.Normal:
 		currentState.resultNormal(alertRule, result)
 	case eval.Alerting:
+		if err := st.takeScreenshot(ctx, alertRule, currentState); err != nil {
+			st.log.Error("failed to take screenshot", "uid", alertRule.UID, "error", err.Error())
+		}
```

Comment:
As discussed, let's make sure we don't do this yet - I would like to discuss this it in a separate PR.